README.md

```diff
@@ -30,3 +30,33 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+When n > 1, $$3T(n/3) + n^5$$ is true.
+
+$$3T(n/3) + n^5$$
+
+$$=3 * 3T(n/(3*3)) + 3(n/3)^5 + n^5$$
+
+$$=3 * 3 * 3T(n/(3 * 3 * 3)) + 3 * 3 * (n/(3 * 3))^5 + 3(n/3)^5 + n^5$$
+
+$$=3 * 3 * 3T(n/(3 * 3 * 3)) + (3 * 3n^5)/((3 * 3)^5) + (3n^5)/(3^5) + n^5$$
+
+$$=3 * 3 * 3T(n/(3 * 3 * 3)) + ((3 * 3)/((3 * 3)^5) + 3/(3^5) + 1) * n^5$$
+
+$$3^i * T(n/(3^i)) + (\sum_{k=0}^{i-1} ((3^k)/(3^{5k})))*n^5$$
+
+Plugging in 1 for i, we get $$3T(n/3) + n^5$$
+
+Next we substitute i with logn
+
+$$3^{logn} * T(n/(3^{logn})) + (\sum_{k=0}^{logn-1} ((3^k)/(3^{5k})))*n^5$$
+
+$$nT(1) + n^5 * ((3^{logn - 1}/(3^{5(logn-1)}) + 1)$$
```

Comment:
I don't see how you get to this line from the previous line. What are you replacing the sum with?

Reply:
It goes away ass the logn-1 is given to the k values after it, and the + 1 is added. I just looked to see if other students are doing it this way, and there are a few.

Reply:
Why does it go away?

Reply:
There's a formula out there that can be used in this instance, where it can be equal to Logn-1. In order for that to work, that's where the 1 comes in, as well as the logn-1 going into the exponents of the other numbers. I don't have a better explanation other than there's a formula for it.

Reply:
And can you please explain that in your solution? In particular you can give the formula.